src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -252,6 +318,324 @@ module Tick : S = struct
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 1
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 1, Next))
+                            + (beta * cell (var (LookupSorted 1, Curr))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 2
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 2, Curr))
+                            + (beta * cell (var (LookupSorted 2, Next))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 3
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 3, Next))
+                            + (beta * cell (var (LookupSorted 3, Curr))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        )
+                    * if_feature
+                        ( LookupsPerRow 4
+                        , (fun () ->
+                            gamma
+                            * ( beta
+                              + field
+                                  "0x0000000000000000000000000000000000000000000000000000000000000001"
+                              )
+                            + cell (var (LookupSorted 4, Curr))
+                            + (beta * cell (var (LookupSorted 4, Next))) )
+                        , fun () ->
+                            field
+                              "0x0000000000000000000000000000000000000000000000000000000000000001"
+                        ) )
+                - cell (var (LookupAggreg, Curr))
```

Comment:
And by subtracting you're telling it to discard the cases where it is Xor, ChaChaFinal, etc, because...? Perhaps a comment with the formula could be super useful